### DISCUSSION TITLE
Import from read-it-later apps

### DISCUSSION THREAD

**Discussion by erlend-sh** [raffomania/linkblocks#36] [Ideas]
I already have thousands of articles stored in my article reader app of choice (Omnivore). In order to effectively use linkblocks, I’d need a way to import selected articles. Preferably on a recurring basis via an API integration as opposed to a full export-import step.

Supporting Pocket plus the two most popular open source options would provide good coverage:

- https://getpocket.com/developer/
- https://docs.omnivore.app/integrations/api.html
- https://developer.raindrop.io/

**Comment by raffomania** [comment]
I'm open to adding this and think integration with the wider ecosystem is excellent, not only with bookmarking services but with other fediverse software as well. Timeline-wise, I'd start working on this after social features and basic federation is completed.
